src/services/exposed/record-updater.js

```diff
@@ -0,0 +1,16 @@
+const AbstractRecordService = require('./abstract-records-service');
+const ResourceDeserializer = require('../../deserializers/resource');
+
+class RecordUpdater extends AbstractRecordService {
+  update(record, recordId) {
+    return new this.Implementation.ResourceUpdater(this.model, { recordId }, record)
+      .perform();
+  }
+
+  deserialize(body) {
+    return new ResourceDeserializer(this.Implementation, this.model, body, false)
```

Comment:
[Misc] Why is there only the update that is done with `withRelationships` = false ?

Reply:
Old codes. I did not change anything on that. But to be short, when you create an entry you can create it with relationships (As you may require a record to have the relationship set to be able to create it). But when you update a record it updates only the record, not the relationships